app/services/katello/event_queue.rb

```diff
@@ -1,15 +1,30 @@
 module Katello
   class EventQueue
+    MAX_AGE = 6.hours
```

Comment:
might make sense to mark "metadata" as serializable. I would prefer key/value style hashes

Reply:
check event.rb - unless you mean something else?

Reply:
my bad :)